service/k8sclient/k8sclient.go

```diff
@@ -586,7 +590,7 @@ func (c *K8sClient) UpdateXtraDBCluster(ctx context.Context, params *XtraDBParam
 func (c *K8sClient) DeleteXtraDBCluster(ctx context.Context, name string) error {
 	res := &pxc.PerconaXtraDBCluster{
 		TypeMeta: common.TypeMeta{
-			APIVersion: pxcAPIVersion,
+			APIVersion: pxcAPINamespace + "/v1",
```

Comment:
could you please check this case:
1. Add DB Cluster
2. Upgrade Operator version
3. Delete DB Cluster

Reply:
I use it only for deleting because it does not matter. It can't be empty and non-existing API. In upgrade docs it's mentioned it should be changed in all db clusters when upgrading operator so we should keep it in `Add DB Cluster` and patch DB clusters on operator upgrade -> #207